PowerShellTools.Test/ScriptDebuggerTest.cs

```diff
@@ -101,7 +106,11 @@ public void ShouldBreakOnBreakPoint()
                                new ScriptBreakpoint(null, fi.FullName, 3, 0, engineEvents.Object)
                            };
 
-            _debugger.SetBreakpoints(sbps);
+            _debugger.BreakpointManager.SetBreakpoints(sbps);

```

Comment:
Do we want a method in the manager that sets & binds them all, as this also exists ~ line 78?